.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to Appium?
+_Put an `x` in the boxes that apply_
+
+- [] Bugfix (non-breaking change which fixes an issue)
```

Comment:
I think all these checkboxes need a space in between the square brackets to render correctly.